cartography/cli.py

```diff
@@ -393,6 +393,39 @@ def _build_parser(self):
                 'The crowdstrike URL, if using self-hosted. Defaults to the public crowdstrike API URL otherwise.'
             ),
         )
+        parser.add_argument(
+            '--rapid7-user',
+            type=str,
+            default=None,
+            help=(
+                'The rapid7 user for authentication.'
+            ),
+        )
+        parser.add_argument(
+            '--rapid7-password-env-var',
+            type=str,
+            default=None,
+            help=(
+                'The name of environment variable containing the rapid7 user password for authentication.'
+            ),
+        )
+        parser.add_argument(
+            '--rapid7-server-url',
+            type=str,
+            default=None,
+            help=(
+                'The url of the Rapid7 InsightsVM server.'
+            ),
+        )
+        # FIXME! not passed correctly
```

Comment:
fixme

Reply:
at this point, I'm not sure why not passed.

Reply:
fixed by using string type for cli argument. not sure why bool not passed correctly.